config/config_to_options.go

```diff
@@ -70,8 +70,10 @@ func (c *AppConfig) addHttpClientOpts(options []engine.ClientOps) []engine.Clien
 
 func (c *AppConfig) addCustomFeeUnit(options []engine.ClientOps) []engine.ClientOps {
 	if c.CustomFeeUnit != nil {
+		santoshis := uint64(c.CustomFeeUnit.Satoshis) //nolint:gosec
```

Comment:
how does it improve anything?

Reply:
My suggestion is to introduce following function to the `bsv` backage.

```golang
// SatoshisFromInt creates a new Satoshis from an integer.
func SatoshisFromInt(s int) (Satoshis, error) {
	if s < 0 {
		return 0, errors.New("value cannot be negative")
	}
	if s > math.MaxUint64 {
		return 0, errors.New("value exceeds maximum uint64")
	}
	// Suppress gosec G115 warning as the conversion is safe due to prior negative check
	return Satoshis(s), nil //nolint:gosec
}
```

And review every single place where this lint-error occurs `integer overflow conversion int -> uint64` and decide whether we want to handle/return additional error or do panic.

To me, `panic` should be for places where "it should never heppen"
and error when the value is provided by a user.